content/en/docs/Kyverno CLI/_index.md

```diff
@@ -115,12 +115,14 @@ kyverno apply /path/to/policy.yaml --resource /path/to/resource.yaml --set <vari
 
 Use `-f` or `--values-file` for applying multiple policies to multiple resources while passing a file containing variables and their values. Variables specified can be of various types include AdmissionReview fields, ConfigMap context data (Kyverno 1.3.6), and API call context data (Kyverno 1.3.6).
 
+Use `-u` or `--userinfo` for applying policies while passing a optional user_info.yaml file which contains necessary admission request data made during the request.
```

Comment:
"while passing an optional..."

Reply:
A question on your code...the file name is not hard-coded to look for `user_info.yaml`, is it?

Reply:
The file name is not hardcoded. The user can pass a cmd flag ```userInfoPath``` to set the file path